src/datasources/network/axios.network.service.ts

```diff
@@ -39,6 +39,17 @@ export class AxiosNetworkService implements INetworkService {
     }
   }
 
+  async delete<T = any, R = NetworkResponse<T>>(
+    url: string,
+    data?: object,
```

Comment:
@fmrsabino @hectorgomezv this is working correctly but in the case of the params an entity was create should I create an entity for data?

Reply:
In my opinion having this defined as `object` is ok, we don't need to map it.